pendingUservoiceCreation/index.ts

```diff
@@ -0,0 +1,44 @@
+import { addDays } from '../shared/utils';
+import { completeFunction } from '../shared/functions';
+import { getAllGitHubIssuesRecursivelyFilterWithLabels, commentGitHubIssue } from '../shared/github';
+
+module.exports = (context) => {
+    const githubApiHeaders = {
+        'User-Agent': 'github-bot-uwp-toolkit',
+        'Authorization': 'token ' + process.env.GITHUB_BOT_UWP_TOOLKIT_ACCESS_TOKEN
+    };
+
+    getAllGitHubIssuesRecursivelyFilterWithLabels(
+        githubApiHeaders,
+        process.env.GITHUB_BOT_UWP_TOOLKIT_REPO_OWNER,
+        process.env.GITHUB_BOT_UWP_TOOLKIT_REPO_NAME,
+        null,
+        ["pending-uservoice-creation"],
+        (issues) => {
+            context.log(`Total of ${issues.length} issues pending uservoice creation.`);
+            context.log(issues);
+
+            const issuesWithoutActivity = issues.filter(issue => {
+                // check if last message was sent x days ago
+                const lastComment = issue.lastComment.edges[0];
+                const today = new Date();
+                const numberOfDaysWithoutActivity = 7;
```

Comment:
Should this use NUMBER_OF_DAYS_WITHOUT_ACTIVITY instead?

Reply:
I asked myself but I am not sure if we should the same variable for two functions. What do you think?